style-guide.md

```diff
@@ -63,10 +63,67 @@ Example:
 ```
 
 
+Documentation comments (dartdocs)
+---------------------------------
+
+We use "dartdoc" for our documentation. All public members in Flutter
```

Comment:
Maybe include a link to https://www.dartlang.org/effective-dart/documentation/#doc-comments?